integrations/pinecone/tests/test_document_store.py

```diff
@@ -257,3 +260,59 @@ def test_embedding_retrieval(self, document_store: PineconeDocumentStore):
         assert len(results) == 2
         assert results[0].content == "Most similar document"
         assert results[1].content == "2nd best document"
+
+    @pytest.mark.skipif("PINECONE_API_KEY" not in os.environ, reason="PINECONE_API_KEY not set")
+    def test_sentence_window_retriever(self, sleep_time):
```

Comment:
I believe this test could be made much shorter if we use the `document_store` fixture, as done for example in `test_write_documents_duplicate_overwrite`.

The fixture should take care of creating and deleting the document store, automatically sleep after writing docs, etc...

Reply:
good point 👍🏽 thanks